app/src/test/java/com/android/sample/model/recipe/RecipeBuilderTest.kt

```diff
@@ -99,6 +99,28 @@ class RecipeBuilderTest {
     assertEquals("5.00", recipe.price)
   }
 
+  @Test
```

Comment:
This looks really good! For the `test addIngredient adds ingredient and measurement()`  just a small suggestion ( I know it's not part of this PR but I reviewed the whole file ) : would it make sense to structure it this way?

```kotlin
val recipe = builder
    .apply {
        setName("Cake")
        setInstructions("Mix all ingredients")
        addIngredientAndMeasurement("Flour", "200g")
        addIngredientAndMeasurement("Sugar", "100g")
    }
    .build()
```

This might feel a bit more logical and keeps the configuration steps together. Just a minor stylistic suggestion—let me know what you think! ( Btw It work)

Reply:
I dont see any inconvenience, to do so